src/db_api_util.ts

```diff
@@ -1031,6 +1052,67 @@ export async function getThreadNeedsAttentionFor(
   }
 }
 
+// Return a list of ThreadInfo's for all threads needing followup
+export async function getThreadsNeedingFollowUp(
+  slackUserId: string,
+  days: number
+): Promise<ThreadInfo[]> {
+  const client = await pool.connect();
+  try {
+    const result = await client.query(
+      `WITH all_claims AS (
+        SELECT
+          user_id
+          , volunteer_slack_user_id
+          , volunteer_slack_user_name
+          , row_number () OVER (PARTITION BY user_id ORDER BY created_at DESC) AS rn
+        FROM volunteer_voter_claims
+        WHERE archived != true
+      ), latest_claims AS (
+        SELECT user_id, volunteer_slack_user_id, volunteer_slack_user_name
+        FROM all_claims
+        WHERE rn = 1
+      ), all_status AS (
+        SELECT
+          user_id, voter_status
+          , row_number() OVER (PARTITION BY user_id ORDER BY created_at DESC) AS rn
+        FROM voter_status_updates
+      ), latest_status AS (
+        SELECT user_id, voter_status
+        FROM all_status
+        WHERE rn = 1
+      )
+      SELECT
+        t.slack_parent_message_ts
+        , t.slack_channel_id
+        , t.user_id
+        , EXTRACT(EPOCH FROM now() - updated_at) as last_update_age
+        , c.volunteer_slack_user_name
+        , s.voter_status
+        FROM threads t, latest_claims c, latest_status s
+        WHERE
+          needs_attention = false
```

Comment:
why needs_attention false?

Reply:
just thinking it was a distinct set from the attention group, but it doesn't need to be that way.. dropping it!